src/routes/products/+page.svelte

```diff
@@ -14,7 +14,7 @@
 
 <section class="container">
   {#each vermouths as vermouth, index}
-    <div in:fade={{ duration: 300, delay: index * 150 }}>
+    <div in:fade|global={{ duration: 300, delay: index * 150 }}>
```

Comment:
pretty sure we should not add this here?

Reply:
This was added only with the migration sure I'm not

Reply:
That is irrelevant, you should know what it does and if it should be there or not

Reply:
In Svelte 3, transitions were global by default and you had to use the |local modifier to make them local. 
if the global transition is not executed

Reply:
Yes I know that, so should this be here or not?

Reply:
now yes

Reply:
why?

Reply:
For the transition to be executed we always set global and if we want to activate it by means of some mechanism we leave it in local before it was in global now we always have to specify it

Reply:
I know what it does, do we need it? Yes or no and why ?

Reply:
Yes, we need it, if not, the transition is not activated because it would be local and we need it global so that it is always executed